MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: Reminders in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the Reminder support in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its element and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Reminder in .NET MAUI Scheduler (SfScheduler)
+
+The MAUI Scheduler notify an appointment reminder by using  the [ReminderAlertOpening] event. An appointment can have one or more reminders.
```

Comment:
The MAUI Scheduler notify an appointment reminder by using the EnableReminder property and [ReminderAlertOpening] event

Reply:
Changes updated.